jbmc/regression/jbmc-concurrency/synchronized-methods/A.java

```diff
@@ -0,0 +1,49 @@
+public class A
+{
+  static int g;
+
+  public synchronized void me1()
+  {
+    g = 0;
+  }
+
+  // expected verification success
+  // --
+  // base-case, no synchronization
+  public void me2()
+  {
+    B t = new B();
+    t.shared = 5;
+    t.start();
+    assert(t.shared == 5 || t.shared == 6);
+  }
+
+  // expected verification success
+  // --
+  // locking mechanism
+  public void me3()
+  {
+    B t = new B();
+    synchronized(t)
+    {
+      t.shared = 5;
+      t.start();
+      assert(t.shared == 5);
+    }
+  }
+}
+
+class B extends Thread
+{
+  public int shared = 0;
+
+  @Override
+  public void run()
+  {
+    set();
+  }
+  public synchronized void set()
+  {
+    shared++;
+  }
+}
```

Comment:
Add a new test with a `static synchronized` method and declare it as `KNOWNBUG`

Reply:
done